### ISSUE TITLE
[Enhancement] 0.25x zoom

### ISSUE THREAD

**Issue by darthvader45** [MamiyaOtaru/voxelmap#3]
@MamiyaOtaru Dunno what version I found this in, but at one point I remember this mod having a 0.25x zoom.  Not sure if that was a temporary feature or I was seeing things, but I'd like to see it in VoxelMap.  Wonder if it could still work.

**Comment by MamiyaOtaru** [comment]
it still does on my end.  0.25 is one of the zoom levels the minimap cycles
through.  The world map can be configured to zoom out farther than that

**Comment by darthvader45** [comment]
Huh, I'm on the latest 1.12.2 version (1.7.1), and it doesn't have it.  Any chance you're running an update that's not out yet?

Edit: Found it under min zoom apparently.  But that's only for world map.  Minimap doesn't have it, unless it's in the config.

**Comment by TylerS1066** [comment]
It depends on the server or client side render distance (not sure which).

**Comment by darthvader45** [comment]
Hmm, so how far would the render distance have to be (in chunks)?

**Comment by TylerS1066** [comment]
I had mine at 10-12 and had the ability to use 0.25 zoom, I'd guess over 8 chunks to have it?

**Comment by darthvader45** [comment]
Hmm, I'll check that out.

**Comment by MamiyaOtaru** [comment]
that is correct. If render distance is at 8 or below, zoom 0.5 shows everything there is to see in the live map.  As the live map only shows chunks that are currently loaded, zooming out farther when the render distance is only 8 chunks will show nothing extra but a blank border around the map
